manifests/init.pp

```diff
@@ -153,6 +153,7 @@
   String                                               $package_name,
   Optional[String]                                     $package_provider,
   String                                               $package_ensure,
+  Boolean                                              $purge_conf_d,
```

Comment:
This needs a default value of `false`

Reply:
Default value set in **data/common.yaml**.

Reply:
I actually think the default should be true at least on Debian, otherwise #223 would still be an issue in the default configuration